.gitattributes

```diff
@@ -0,0 +1 @@
+**/*zz_generated_*.go linguist-generated=true
```

Comment:
> **Note** This will indicate to GitHub ([docs](https://docs.github.com/en/repositories/working-with-files/managing-files/customizing-how-changed-files-appear-on-github)) that `zz_generated_*.go` files are generated and so they aren't loaded by default.